sample/src/main/kotlin/com/thefuntasty/donutsample/ui/playground/PlaygroundActivity.kt

```diff
@@ -0,0 +1,208 @@
+package com.thefuntasty.donutsample.ui.playground
+
+import android.os.Bundle
+import android.os.Handler
+import android.widget.SeekBar
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.thefuntasty.donut.DonutDataset
+import com.thefuntasty.donutsample.R
+import com.thefuntasty.donutsample.data.model.BlackCategory
+import com.thefuntasty.donutsample.data.model.DataCategory
+import com.thefuntasty.donutsample.data.model.GreenCategory
+import com.thefuntasty.donutsample.data.model.OrangeCategory
+import com.thefuntasty.donutsample.tools.extensions.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/a83effedb527357706bc3f0552e1f26e737d3fae/sample/src/main/kotlin/com/thefuntasty/donutsample/ui/playground/PlaygroundActivity.kt#L14"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>